core/src/test/scala/org/apache/spark/ui/StagePageSuite.scala

```diff
@@ -84,6 +84,19 @@ class StagePageSuite extends SparkFunSuite with LocalSparkContext {
         jobListener.onTaskEnd(
           SparkListenerTaskEnd(0, 0, "result", Success, taskInfo, taskMetrics))
     }
+    (3 to 4).foreach {
```

Comment:
You are right. I did try to simulate a similar unit test for Executors Tab UI, but it needs creating a SparkUI object which cannot be accessed from StagePageSuite. Hence, I am reverting my unit test for the time being.